### DISCUSSION TITLE
Spam user comments on Pubs

### DISCUSSION THREAD

**Discussion by hlj2014** [knowledgefutures/pubpub#2969] [Support]
Related to #1498:

What is the best way to report a user I believe to be posting spam comments within a community? We received comments on 4 different pubs, links below, from [this user](https://www.pubpub.org/user/terry-thomas) within a 5 minute period on January 3, 2024. Each comment contains links to different “services.” Is it possible to block an individual user from a community? Can you please advise?
 
https://handbook.pubpub.org/pub/assignment-house-museum/release/1#discussion-fe4ceb25-3511-434f-a24a-36e6a9a7eb73
 
https://handbook.pubpub.org/pub/case-vmlk/release/1#discussion-80885e47-1753-4544-8b42-c7ceef14845b
 
https://handbook.pubpub.org/pub/case-scribes-geniza/release/2#discussion-f0ed1e74-4adb-4813-92f3-2c7688a5bf10
 
https://handbook.pubpub.org/pub/case-visualizing-alchemy/release/1#discussion-dc9439d8-c035-4b5f-9bef-4ffac6d02e47
 
Thank you!

**Comment by meeloun** [comment]
When choosing undergraduate online courses      [http://www.wangkedaixiu.com/wkds/bk/](http://www.wangkedaixiu.com/wkds/bk/)         as a substitute service, one needs to pay attention to multiple aspects to ensure that their choice is correct. By carefully identifying and communicating with institutions, it can help you find personalized services that truly meet your needs.
